tests/selfReportToIWF.test.ts

```diff
@@ -103,7 +103,28 @@ describe('selfReportToIWF', () => {
       user_age_range: '<13',
     };
 
-    await selfReportToIWF({ ...baseContext }, event, () => {});
+    // @ts-ignore
+    axios.mockImplementationOnce(async () => ({
+      data: {
+        result: 'OK',
+        message: {
+          access_token: 'SECRET TOKEN',
+        },
+      },
+    }));
+
+    const callback: ServerlessCallback = (err, result) => {
```

Comment:
As I said in the other PR, this approach is slightly flawed because it passes if the callback never gets called.

If we make the callback a `jest.fn()`, we can verify they gets called at all using `expect(...).toHaveBeenCalled()`, then run detailed assertions on the parameters by pulling them from the `mocks` property: https://jestjs.io/docs/mock-function-api#mockfnmockcalls (because `expect(...).toHaveBeenCalledWith(...)` can be limiting.

Generally, any assertion checks that gets executed conditionally should be considered a smell in testing code